microsoft-edge/webview2/how-to/webresourcerequested.md

```diff
@@ -0,0 +1,582 @@
+
+A [request header](https://developer.mozilla.org/docs/Glossary/Request_header) can be used to indicate the format of the response (such as the `Accept-*` headers), set authentication tokens, read and write cookies (sensitive information), modify the user agent, and so on.  A [response header](https://developer.mozilla.org/docs/Glossary/Response_header) can be used to provide more context of the message being sent. 
+
+### Filtering the WebResourceRequested event based on URL and resource type
+
+In a `WebResourceRequested` event, you can specify a filter for the requests that the app is interested in based on URL and resource type.  If the host app uses a filter, the filter must be added before a `WebResourceRequested` event is fired.
+
+For example, suppose the host app is trying to replace images.  In this case, the host app is only interested in web resource requested events for images.  The host app would only get events for images by specifying the filter for images.  Another example is if the host app is only interested in all requests that are under a domain name like `https://example.com`, then the app can use the URL filter to get events that are associated with that site.
+
+# [.NET](#tab/dotnet)
+
+* [CoreWebView2.AddWebResourceRequestedFilter Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.addwebresourcerequestedfilter)
+
+# [Win32](#tab/win32)
+
+* [AddWebResourceRequestedFilter method](/microsoft-edge/webview2/reference/win32/icorewebview2#addwebresourcerequestedfilter)
+
+---
+
+
+For details about how the URL filter works, see [CoreWebView2.AddWebResourceRequestedFilter Method > Remarks](/dotnet/api/microsoft.web.webview2.core.corewebview2.addwebresourcerequestedfilter#remarks)
+
+
+### Why would you want to intercept requests that are sent from WebView2?  
+
+Intercepting requests sent from WebView2 enables you to further configure your request. The host app might want to provide optional content as part of the request that the WebView2 control won't know on its own. Some scenarios include:
+*  You're logging into a page and the app has credentials so the app can provide authentication header without the user having to enter those credentials.  
+*  You want offline functionality in the app so you redirect the URL to a local file path when no internet connection is detected.
+*  You want to upload local file content to the request server via a POST request.
+
+
+### Sequence for modifying requests
+
+<!-- wiki page that points to the Visio source file: Documentation > "Notes about specific image files" -->
+![Diagram of sequence for modifying requests.](webresourcerequested-images/sequence-for-modifying-requests.png)
+
+1. The host app sets up a `WebResourceRequested` filter.
+1. The host app defines the event handlers for `WebResourceRequested` and `WebResourceResponseReceived`.

```

Comment:
Please fix the ordering numbers here.

Reply:
@yildirimcagri - are the rendered numbers correct?  
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/how-to/webresourcerequested?branch=pr-en-us-1808&tabs=dotnet#sequence-for-modifying-requests

The Markdown tagging convention in our repo is all 1's.  The Contributor Guide says you "can" do it using all 1's: https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=main#numbered-list